### PR TITLE
Feature/url param filtering

### PR DESCRIPTION
**Pull request by tuckerbuchy**
@lkysow This is the changes for the filters being added

### CODE REVIEW

File: src/angular/Services/serviceslist.js [hunk 1]
```diff
@@ -43,7 +43,8 @@ services.factory('ServicesList', ['$http', '$translate', 'PopupBuilder', 'Cookie
 
                             // when a user clicks on a map marker, show the service in the sidebar
                             serviceMarker.on('click', function() {
-                                window.location = '/#/services/' + feature.id + '?hideOthers=false';
+                                var parameters = $location.search();
+                                $location.path('services/'+feature.id).search(parameters);
```
Comment (lkysow) [inline src/angular/Services/serviceslist.js:47]: Still need to add `hideOthers=false` here

File: src/angular/controllers/map.controller.js [hunk 2]
```diff
@@ -3,6 +3,20 @@ var controllers = angular.module('controllers');
 controllers.controller('MapCtrl', ['$scope', '$rootScope', '$location', '$translate', 'Search', function ($scope, $rootScope, $location, $translate, Search) {
     var mapCtrl = this;
 
+    mapCtrl.onRegionClick = function(event) {
```
Comment (lkysow) [inline src/angular/controllers/map.controller.js:6]: I think we can delete this, we're not going to allow clicking on regions anymore.

**Comment by tuckerbuchy** [comment]
@lkysow Just realized that I forgot to do Regions (I got region by id)

**Comment by lkysow** [comment]
@tuckerbuchy that's cool, I made some changes to get it working.

**Pull request merged**